### PR TITLE
Add context info the navbar as breadcrumbs

### PR DESCRIPTION
**Pull request by mskrip**

### CODE REVIEW

**Comment by mitas1** [comment]
Sign out shoud be in appbar
screen shot 2017-12-18 at 00 22 11

**Comment by mskrip** [comment]
Fixed

File: src/buildinglog/views/customers.py [hunk 1]
```diff
@@ -18,21 +18,38 @@ def get_or_none(model, *args, **kwargs):
...
+    }]
+
     customers = Customer.objects.all()
 
     notifications = []
     if request.session.get('new_item', False):
         notifications.append('New customer created')
         request.session['new_item'] = False
 
+    print(request.build_absolute_uri('/'))
```
Comment (mitas1) [inline src/buildinglog/views/customers.py]: why this `print`?

Comment (mskrip) [reply src/buildinglog/views/customers.py]: Removed

**Comment by mitas1** [comment]
LGTM

**Pull request merged**